haskell/signable-haskell-protoc/src/Proto/Google/Protobuf/Compiler/Plugin_Fields.hs

```diff
@@ -55,6 +55,12 @@ fileToGenerate ::
    Data.ProtoLens.Field.HasField s "fileToGenerate" a) =>
   Lens.Family2.LensLike' f s a
 fileToGenerate = Data.ProtoLens.Field.field @"fileToGenerate"
+generatedCodeInfo ::
+  forall f s a.
+  (Prelude.Functor f,
+   Data.ProtoLens.Field.HasField s "generatedCodeInfo" a) =>
+  Lens.Family2.LensLike' f s a
+generatedCodeInfo = Data.ProtoLens.Field.field @"generatedCodeInfo"
```

Comment:
From where this comes? Is it auto-generated, or you upgraded it manually somehow?

Reply:
autogenerated